charts/kube2iam/README.md

```diff
@@ -21,9 +21,8 @@ This chart bootstraps a [kube2iam](https://github.com/jtblin/kube2iam) deploymen
 To install the chart with the release name `my-release`:
 
 ```console
-$ helm repo add kube2iam https://buildkite.github.io/charts/
-$ helm repo update
-$ helm install stable/kube2iam --name my-release
+$ helm repo add kube2iam https://kube2iam.github.io/charts/
```

Comment:
This is probably going to be http://jtblin.github.io/kube2iam/. Can someone verify?

Reply:
I tested with `helm fetch kube2iam --repo http://jtblin.github.io/kube2iam` and it seemed to pull the chart down (v`2.5.2`) just fine.

Reply:
Could we please bump the chart version? Since there is a `2.5.3` already on the (now read-only/deprecated) stable repo perhaps `2.5.4` or `2.6.0` is appropriate given the repo move?